### ISSUE TITLE
Cannot build 1 of 3 apps after upgrading to NX 0.54 and app level tsconfig files

### ISSUE THREAD

**Issue by thaoula** [nrwl/nx#169]
Hi Guys,

Got a very strange situation where after upgrading to 0.54 and moving to app based tsconfig files we are unable to AOT build 1 app from 3.

We can build the APP without AOT and it builds fine. However, when building the APP with --prod the build fails whilst reporting several components in a different app not being part of an Ngmodule. 

The components that are broken belong to the AppModule of a different app. The weirdest thing is that the error is not for all the components in the other AppModule its for only 4/6.

So in summary, 

- We are using Angular 5.1, Angular CLI (release16), NX 0.54. 
- The tsconfig.app are the defaults created by nx-migrate.
- All apps except the one can be built using AOT.
- All apps can build without AOT
- We can build the app using AOT if we revert the app back to using the parent tsconfig.

We have been trying to track down this issue for 4 days. Ran out of ideas and was wondering if someone has seen something like this before.

Regards,
Tarek

**Comment by thaoula** [comment]
Guys,

I think this is user error. We found an incorrect import.

Regards,
Tarek
